serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc

```diff
@@ -24,90 +22,93 @@ kubernetes:<group>/<version>/<kind>/<namespace>/<resourceName>?<attributeName>=<
                                                                    - labels=label-name=label-value;other-label=other-value
 ----
 
-The following scheme values are supported:
+Therefore, you must keep in mind that if application startup time matters, then consider to use a static URL instead. 
 
-* **kubernetes**
-* **openshift**
-* **knative**
+The following scheme values are supported in the URI pattern:
 
+* `kubernetes`
+* `openshift`
+* `knative`
 
-For the Kubernetes GVK, the following resources are currently supported:
+The following resources are supported for the Kubernetes GVK (Group, Version, and Kind):
 
-* **v1/service**
-* **serving.knative.dev/v1/service**
-* **v1/pod**
-* **apps/v1/deployment**
-* **apps.openshift.io/v1/deploymentconfig**
-* **apps/v1/statefulset**
-* **route.openshift.io/v1/route**
-* **networking.k8s.io/v1/ingress**
+* `v1/service`
+* `serving.knative.dev/v1/service`
+* `v1/pod`
+* `apps/v1/deployment`
+* `apps.openshift.io/v1/deploymentconfig`
+* `apps/v1/statefulset`
+* `route.openshift.io/v1/route`
+* `networking.k8s.io/v1/ingress`
 
 [IMPORTANT]
 ====
-The **Kubernetes GVK** (Group, Version and Kind) is **mandatory** and can't be empty, otherwise,
-the engine will throw a validation issue and the discovery will not be performed. The last value in the URI
-will always be considered as **Resource Name**.
-The **Namespace** is **optional**. In case of an empty value, the current namespace or context will be used.
-====
+The Kubernetes GVK is mandatory and cannot be empty. In case the Kubernetes GVK is empty, the engine throws a validation issue and discovery is not performed. The last value in the URI is always considered as resource name. 
 
-== Additional Query Parameters
+The `namespace` in the URI is optional, however, if `namespace` contains an empty value, the current namespace or context is used.
+====
 
-Also known as Query String, they are defined in the very same way that we are already used to do with URLs to assign value
-to specific attributes.
+Query parameters in URI::
```

Comment:
```suggestion
KubeURI Query Parameters::
```

Reply:
Same thing. This is just a URI that represents a Kubernetes object in the cluster.

Reply:
So, should we keep as-is?